mmv1/third_party/terraform/services/clouddeploy/resource_clouddeploy_automation_test.go.erb

```diff
@@ -0,0 +1,117 @@
+<% autogen_exception -%>
```

Comment:
This file will need a version guard per https://googlecloudplatform.github.io/magic-modules/develop/test/ (under "Handwritten")

That'll resolve the destroyproducer error- it's currently (correctly) only present at beta+, along with the resource implementation.